BLSQ/iaso
# POLIO-1533 Remove vials used from Form A add edit and list

Remove "vials used" from Form A add/edit/list in Vaccine Stock Management

Related JIRA tickets : POLIO-1533

## Self proofreading checklist


## Changes

Remove the field vials used from Add/ Edit modal of Form A in stock variations.
Remove the column in the list
Made field optional in django model

## How to test

Go to Stock Management and add or edit a Form A and see if the field is visible or not
